queue/python/queue.py

```diff
@@ -0,0 +1,32 @@
+class Queue:
+    def __init__(self):
+        self.items = []
+
+    def isEmpty(self):
+        return self.items == []
+
+    def enqueue(self, item):
+        self.items.insert(0,item)
+
+    def dequeue(self):
+        return self.items.pop()
+
+    def size(self):
+        return len(self.items)
+    
+    def front(self):
+        return self.items[self.size()-1] 
+
+def main():
+    q=Queue()
+    q.enqueue(4)
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpvd7j5r87/queue/python/queue.py
+++ b/tmp/tmpvd7j5r87/queue/python/queue.py
@@ -24,7 +24,7 @@
     q.enqueue(True)
     print("Size of queue is : " + str(q.size()))
     print("Element on front is : " + str(q.front()))
-    print (str(q.front()) + " is Deleted ")
+    print(str(q.front()) + " is Deleted ")
     q.dequeue()
 
 
```